src/main/resources/liquibase/harness/change/expectedSnapshot/edb-e/9.6/alterSequence.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
can liquibase get more details about sequences for edb<10? This json doesn't actually validate if sequence was altered - like `incrementBy` `cycle` or other fields changed something. Default (liquibase/harness/change/expectedSnapshot/alterSequence.json) does more precise checks

Reply:
I'll need to take a closer look at this issue. Perhaps additional core functionality can be added.

Reply:
After discussion at standup today, we will go with `edb-edb` for EDB with the EDB driver and `edb-postgres` for EDB with the postgres driver. CC @yodzhubeiskyi